grape/Gemfile

```diff
@@ -1,8 +1,17 @@
 source 'https://rubygems.org'
 
 gem 'grape'
+gem 'grape-entity'
```

Comment:
please lock the version for each gem, it is safer for everybody (like when we lock the ruby version)